Services/COPage/classes/class.ilPCVerification.php

```diff
@@ -44,7 +45,7 @@ public function setNode($a_node)
     * @param	object	$a_pg_obj		Page Object
     * @param	string	$a_hier_id		Hierarchical ID
     */
-    public function create(&$a_pg_obj, $a_hier_id, $a_pc_id = "")
+    public function create(object $a_pg_obj, string $a_hier_id, $a_pc_id = "") : void
```

Comment:
The 3rd parameter still has no typehint.

Reply:
Could you specifiy the first type more precisely? Maybe you get an idea if you execute this code while testing it in the user interface. You could use `Xdebug`.